src/canisters/ledger/ledger.request.converts.ts

```diff
@@ -3,15 +3,13 @@ import {
   TransferArgs as TransferRawRequest,
 } from "../../../candid/ledgerTypes";
 import { ICPTs, Subaccount } from "../../../proto/ledger_pb";
-import {
-  SUB_ACCOUNT_BYTE_LENGTH,
-  TRANSACTION_FEE,
-} from "../../constants/constants";
+import { TRANSACTION_FEE } from "../../constants/constants";
 import { TransferRequest } from "../../types/ledger_converters";
-import { numberToArrayBuffer } from "../../utils/converter.utils";
 
-export const subAccountIdToSubaccount = (subAccountId: number): Subaccount => {
-  const bytes = numberToArrayBuffer(subAccountId, SUB_ACCOUNT_BYTE_LENGTH);
+export const subAccountNumbersToSubaccount = (
+  subAccountNumbers: number[]
+): Subaccount => {
+  const bytes = new Uint8Array(subAccountNumbers).buffer;
```

Comment:
nth: `const {buffer} = ...` would work right?

Reply:
I can fix it in another PR

Reply:
one of these days, nth